src/ast/codegen_llvm.cpp

```diff
@@ -181,13 +181,13 @@ void CodegenLLVM::visit(Builtin &builtin)
       offset = arch::arg_offset(arg_num);
     }
 
+    Value *ctx = b_.CreatePointerCast(ctx_, b_.getInt64Ty()->getPointerTo());
```

Comment:
I wonder if we can't just do this when `ctx_` is assigned (grep for `ctx_ = `)

Reply:
I eventually want to do this (also see https://github.com/iovisor/bpftrace/pull/1104#discussion_r371649994). I'll do it as a follow up PR once the current ones are in. It should be easier to review and spot issues when all the other fixes have been applied